app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,37 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+body {
+  font-family: 'helvetica';
+}
+
+header {
+  height: 50px;
+  background-color: skyblue;
```

Comment:
Color `skyblue` should be written in hexadecimal form as `#87ceeb`<br>Color literals like `skyblue` should only be used in variable declarations; they should be referred to via variable everywhere else.